packages/injectable/src/dependency-injection-container/createContainer.registration.test.js

```diff
@@ -15,6 +16,24 @@ describe('createContainer.registration', () => {
     expect(actual).toBe('some-injected-instance');
   });
 
+  it('throws an error with the key if the value behind has no default export', () => {
+    const context = new Map();
+
+    context.set("./foobar.injectable.ts", {
+      notDefault: getInjectable({
+        id: 'irrelevant',
+        instantiate: () => 'some-injected-instance',
+      }),
+    });
+
+    expect(() => createContainer(() => Object.assign(
+      (key) => context.get(key),
+      {
+        keys: () => context.keys(),
+      }
+    ))).toThrowError(/\.\/foobar\.injectable\.ts/);
```

Comment:
A tad too complicated test setup distracts from what is being tested. Suggestion written blindly.
```suggestion
    const requireContextStub = Object.assign(
      () => ({
        notDefault: 'irrelevant' 
      }),
      
      {
        keys: () => ['./some.injectable.js'],
      }
    );

    expect(() => createContainer(() => requireContextStub)).toThrowError('File "./some.injectable.js" does not have a default export.');
```